xml/obs_build_job_constraints.xml

```diff
@@ -149,6 +149,20 @@ Constraint: hardware:cpu:flag sse2</screen>
     <para>Example for project configuration:</para>
     <screen>Constraint: hardware:processors 4</screen>
   </sect2>
+  <sect2>
+   <title><literal>jobs</literal></title>
+   <para> To require a minimal number of pre-confiured parallel jobs for the build job. 
+          This limits the parallel wanted jobs for the build tooling, but is actually not
+          a hardware requirement.</para>
+   <para> Example for _constraints file:</para>
+    <screen>&lt;constraints&gt;
+  &lt;hardware&gt;
+    &lt;jobs&gt;4&lt;/jobs&gt;
+  &lt;/hardware&gt;
+&lt;/constraints&gt;</screen>
```

Comment:
Just a hint: for XML it's enough to mask the `<` character with `&lt;`. However, this is not necessary for `>`. It's allowed, but you don't have to. Saving a few characters may help. :wink: